pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -146,6 +170,19 @@ def add_oa_cuts(target_model, dual_values, solve_data, config,
 
 
 def add_int_cut(var_values, solve_data, config, feasible=False):
+    """
+    Modifies the model by adding integer cuts (if needed)
```

Comment:
I'd remove the (if needed) since the function adds the cut if it is called. Try using the same template for the description as `add_oa_cut`